### DISCUSSION TITLE
Could the developers of Open-Shell make it work perfectly well with Windows 11. Thanks.

### DISCUSSION THREAD

**Discussion by MichaelIsGreat123** [Open-Shell/Open-Shell-Menu#762] [General]
Please, to the developers of Open-Shell, could you update it to make sure it works as seamlessly as it works on Windows 10. Thanks. I have never seen such a bunch of idiots at Microsoft with the new (very bad) Start Menu on Windows 11!!!

**Comment by Gaurav-Original-ClassicShellTester** [comment]
I feel that currently, ExplorerPatcher is really is the best way to bring back classic Taskbar and make everything else work correctly, including Open Shell/Classic Shell and without breaking the new functionality of Windows 11: https://github.com/valinet/ExplorerPatcher ExplorerPatcher brings back the classic Taskbar without the side effects of UndockingDisabled.

**Comment by MichaelIsGreat123** [comment]
Hello Tester,


Thanks for your feedback. But it does not seem that ExplorerPatcher provides the same Start Menu GUIs that were available with Open-Shell?


Kind Regards
Michael
-----------------------------------------------
***@***.***>
________________________________

**Comment by Gaurav-Original-ClassicShellTester** [comment]
It enables the Classic Taskbar after which you can install Open Shell and it will work just like it worked with Windows 7/8/10.

**Comment by MichaelIsGreat123** [comment]
Hello Tester,


Thanks for your feedback. Just a remark. What I really liked about Open-Shell under Windows 10 is that you had both, Open-Shell but you can also use the Windows 10 default Start Menu if you want using Ctrl+Shift+Start Menu. With this Explorer Patcher, you cannot use the default system anymore, you have to kill the default system first with this Explorer Patcher and then install Open-Shell for it to be able to work. It is kind of disappointing I could say. But it still better than no solution.

Honestly, Microsoft, they are really bunch of idiots with their UI. They could have given choices to users but instead, in order to keep it simple, they make it completely unproductive to use.


Kind Regards
Michael

-------------------------------------------------------
***@***.***>
________________________________

**Comment by Gaurav-Original-ClassicShellTester** [comment]
That's incorrect. You can use the default Windows 11 Start menu with ExplorerPatcher besides Open Shell. Nothing has to be killed with ExplorerPatcher. However Microsoft deleted the Windows 10 Start menu from Windows 11.

**Comment by Ibuprophen** [comment]
Thank you very much for the input and the link that I plan to try out when I eventually obtain & install Windows 11 on one of my test computers. It looks very promising. 

The title of this discussion kinda throws you off as opposed to the content. Just a suggestion... :-)) 

After looking into that link a bit, I (personally) don't believe that it's something that we would implement in Open Shell as a majority of it addresses something that this software doesn't really get into. 

It's possible (as I haven't tried it yet) that one may use that type of Windows 11 patch before installing the Open Shell software. It "may/might" help Open Shell to perform a bit better. As already stated, I can't personally state this for sure till I can test it out myself. 

Anyways, thanks for the input and heads up. :-)) 


~Ibuprophen

**Reply by Gaurav-Original-ClassicShellTester** [reply]
Yes I don't think either that it's ideal for integrating *into* Open Shell especially given how Windows-11-as-a-Service changes. Just something to install beforehand that'll make everything work better, including Open Shell but also other features like tray icons, and the taskbar as a whole (since it activates the working taskbar code still left intact) - which btw Microsoft can remove at any point.
